src/components/TransactionHistoryAddressAndID/address.js

```diff
@@ -0,0 +1,31 @@
+import React, { PropTypes } from 'react';
+import { TableCell } from 'material-ui/Table';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import { withStyles } from 'material-ui/styles';
+
+import styles from './styles';
+
+class TransactionHistoryAddress extends React.PureComponent {
```

Comment:
This should be a stateless component if we're not using state.

const TransactionHistoryAddress = ({ classes, transaction }) => (
  ...
)